doc/optimizers.md

```diff
@@ -2886,3 +2886,56 @@ optimizer.Optimize(f, coordinates);
  * [SGD](#standard-sgd)
  * [SGD in Wikipedia](https://en.wikipedia.org/wiki/Stochastic_gradient_descent)
  * [Differentiable separable functions](#differentiable-separable-functions)
+
+## Yogi
+
+*An optimizer for [differentiable separable functions](#differentiable-separable-functions).*
+
+Yogi is an optimization algorithm based on Adam with more fine grained
+effective learning rate control, and has similar theoretical guarantees on
+convergence as Adam.
```

Comment:
Maybe it is worth mentioning that Yogi uses additive updates instead of multiplicative updates for the `v` parameter?